core/domain/html_cleaner.py

```diff
@@ -182,7 +182,13 @@ def unescape_html(escaped_html_data):
     Returns:
         str. Unescaped HTML string.
     """
-    unescaped_html_data = escaped_html_data
+    # Some of the explorations having font-family in style tags
+    # include double quotes around the font name. This results
+    # in error while performing json.loads.
+    # Any such extra quotes are not rendered as \&quot; in html
+    # string. They are rendered in form of &amp;quot;. This line
```

Comment:
Interesting, what does this content look like in the frontend? The frontend has to do a similar json conversion process, so it seems like that would error due to the extra double quotes.

What will your change  (removing the extra double quotes) do, visually? Won't simply removing the double quotes make the style tag invalid?

Reply:
The comment can be simplified:

Some content erroneously contains un-escaped double quotes (`&amp;quot;` instead of `\&amp;quot;`) which breaks json parsing. This line ensures all double quotes are escaped.